clustered/server/src/test/java/org/ehcache/clustered/server/store/ClusterTierActiveEntityTest.java

```diff
@@ -877,6 +877,7 @@ public void testSyncToPassiveBatchedByDefault() throws Exception {
 
   @Test
   public void testDataSyncToPassiveCustomBatchSize() throws Exception {
+    System.setProperty(ClusterTierActiveEntity.SYNC_DATA_SIZE_PROP, "512");
```

Comment:
You need to unset this in a `finally` otherwise it will stick for ever after

Reply:
Good find. This was always like that. I just moved this statement a few lines above.

But it is a must fix.. I will do it,

Reply:
@henri-tremblay Will fix this..

Actually the clear property was already there at the end of the test..So it is actually not all that bad.  It was my bad that I did not move the try { when I moved the set property up, But behaviour wise was everything looks ok and the other tests were running without this property 

Anyway, the key point is tests was running properly all this time as the clearProperty was there at the end.